Update team members documentation (#173)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/team-members.mdx

> -> **Note:** To add users to a team, they must first be invited to join the organization by email, and accept the invitation. This ensures that you don't accidentally add the wrong person by mistyping a username. See [the Organization Memberships API documentation](/cloud-docs/api-docs/organization-memberships) for more information.

Reviewer: ```suggestion
-> **Note:** To add users to a team, they must first receive and accept the invitation to join the organization by email. This process ensures that you do not accidentally add the wrong person by mistyping a username. Refer to [the Organization Memberships API documentation](/cloud-docs/api-docs/organization-memberships) for more information.
```

Suggest rewording the 1st sentence to avoid a passive voice style issue. Suggest adding "process" to avoid a "This" without a subject in the 2nd sentence. Change "See" to "Refer to" to avoid ableist language.

  https://app.terraform.io/api/v2/teams/257525/relationships/users
```

## Add a User to Team (With organization membership ID)

> This method adds multiple users to a team using the organization membership ID.

Reviewer 2: ```suggestion
This method adds multiple users to a team using the organization membership ID. Unlike the user ID method, the user doesn't need to exist yet; they only need to have been invited to the organization.
```

| Parameter  | Description         |
| ---------- | ------------------- |
| `:team_id` | The ID of the team. |

### Request Body

This POST endpoint requires a JSON object with the following properties as a request payload.

Properties without a default value are required.

| Key path      | Type   | Default | Description                      |
| ------------- | ------ | ------- | -------------------------------- |
| `data[].type` | string |         | Must be `"organization-memberships"`.               |
| `data[].id`   | string |         | The organization membership ID of the user to add. |

### Sample Payload

```json
{
  "data": [
    {
      "type": "organization-memberships",
      "id": "ou-nX7inDHhmC3quYgy"
    },
    {
      "type": "organization-memberships",
      "id": "ou-tTJph1AQVK5ZmdND"
    }
  ]
}
```

### Sample Request

```shell
curl \
  --header "Authorization: Bearer $TOKEN" \
  --header "Content-Type: application/vnd.api+json" \
  --request POST \
  --data @payload.json \
  https://app.terraform.io/api/v2/teams/257525/relationships/organization-memberships
```

## Delete a User from Team (With user ID)

> This method removes multiple users from a team using the user ID. Both users and teams must already exist. This DOES NOT delete the user; it only removes them from this team.

Reviewer: ```suggestion
This method removes multiple users from a team using the user ID. Both users and teams must already exist. This method only removes a user from this team. It does not delete that user overall.
```

Suggest rewording to adding "method" after "This" to give "This" a subject. Change unneeded capitalization.

  --data @payload.json \
  https://app.terraform.io/api/v2/teams/257525/relationships/users
```

## Delete a User from Team (With organization membership ID)

> This method removes multiple users from a team using the organization membership ID. This DOES NOT delete the user; it only removes them from this team.

Reviewer: ```suggestion
This method removes multiple users from a team using the organization membership ID. This method only removes a user from this team. It does not delete that user overall.
```

Suggest rewording to adding "method" after "This" to give "This" a subject. Change unneeded capitalization.
